kore/src/Kore/Repl.hs

```diff
@@ -6,64 +6,428 @@ License     : NCSA
 Maintainer  : vladimir.ciobanu@runtimeverification.com
 -}
 
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-unused-top-binds    #-}
```

Comment:
I strongly dislike turning off warnings by-module. I think it would be appropriate to export `ReplState`; that would encourage developing modular REPL features, and remove the warning.